db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> None:
+        print(f"{self.name}")
```

Comment:
You should place a `return` statement, not print in `__str__`. Indicate this in type annotation and do not overuse the f-strings, there is no need for them when you return `CharField` directly.

Reply:
Which annotation type i should put here? Not None?